packages/manager/apps/pci-ai-tools/src/pages/notebooks/[notebookId]/dashboard/_components/Resources.component.tsx

```diff
@@ -0,0 +1,74 @@
+import { HelpCircle } from 'lucide-react';
+import { useTranslation } from 'react-i18next';
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  Slider,
+} from '@datatr-ux/uxlib';
+import { useNotebookData } from '../../Notebook.context';
+import { bytesConverter } from '@/lib/bytesHelper';
+import ResourcesSpec from '@/components/resources-spec/ResourcesSpec.component';
+import { isStoppedNotebook } from '@/lib/statusHelper';
+
+const Resources = () => {
+  const { notebook } = useNotebookData();
+  const { t } = useTranslation('ai-tools/notebooks/notebook/dashboard');
+  return (
+    <div data-testid="resources-container">
+      <ResourcesSpec
+        resources={notebook.spec.resources}
+        allowUpdate={true}
+        disabled={!isStoppedNotebook(notebook.status.state)}
+      />
+      <div className="flex flex-row items-center gap-2">
+        <span>
+          {t('workspaceStorage', {
+            storage: bytesConverter(
+              notebook.status.workspace.storageFree,
+              false,
+              1,
+            ),
+          })}
+        </span>
+        <Popover>
+          <PopoverTrigger>
+            <HelpCircle className="size-4" />
+          </PopoverTrigger>
+          <PopoverContent>
+            <p>{t('workspaceStorageHelper')}</p>
+          </PopoverContent>
+        </Popover>
+      </div>
+      <div className="flex flex-col gap-1 mt-3 mr-4">
+        <Slider
+          className="[&_[role=slider]]:hidden"
+          data-testid="storage-slider"
+          id="storage-slider"
+          name="storage-slider"
+          defaultValue={[0]}
+          disabled={true}
+          value={[notebook.status.workspace.storageUsed]}
+          min={0}
+          max={notebook.status.workspace.storageFree}
+        />
```

Comment:
You may want to use a `Progress` component which is only a display component, instead of the form element `Slider`. This would help with accessibility:

```suggestion
       <Progress value={notebook.status.workspace.storageUsed * 100 / notebook.status.workspace.storageFree} />
```
<Progress value={33} />

Reply:
Done :)